app.js

```diff
@@ -72,6 +78,10 @@ app.get('/deny', function(req, res) {
 	res.send("Denied request");
 });
 
+app.get('/add_directory', function(req, res) {
```

Comment:
Should these directories be underscores or dashes? I'm in favor of the latter just for aesthetic reasons. (Super low priority)